Add section on external charts (#5963)
istio/istio.io: content/en/docs/setup/install/istioctl/index.md

$ istioctl manifest apply --set values.global.mtls.enabled=true --set values.global.controlPlaneSecurityEnabled=true
{{< /text >}}

> In general, you can use the `--set` flag in `istioctl` as you would with `Helm`. The major difference is you must

Reviewer: ```suggestion
In general, you can use the `--set` flag in `istioctl` as you would with
[Helm](https://github.com/helm/helm). The only difference is you must
```

> prefix the `Helm` paths with `values.` because this is the path to the `Helm` pass-through API.

Reviewer: ```suggestion
prefix the setting paths with `values.` because this is the path to the Helm pass-through API, described below.
```

## Install from external charts

By default, `istioctl` uses compiled-in charts to generate the install manifest. These charts are released together with
`istioctl` for auditing and customization purposes and can be found in the release tar in the
`install/kubernetes/operator/charts` directory.

> `Istioctl` can also use external charts rather than the compiled-in ones. To select external charts, set

Reviewer 2: ```suggestion
`istioctl` can also use external charts rather than the compiled-in ones. To select external charts, set
```

Author: afaics capitalizing symbols is the norm in the docs.

Reviewer 2: I don't think grammar rules apply to things in `code blocks`, but will let docs owners decide

Reviewer: Other than in headings, we generally prefer to keep commands in lower case, even at the beginning of a sentence, which isn't ideal. The alternative is to change the sentence to:
```
The `istioctl` command can also use ...
```

`installPackagePath` to a local filesystem path:

{{< text bash >}}

> $ istioctl manifest apply --set installPackagePath=~/istio-releases/istio-1.4.0/install/kubernetes/operator/charts

Reviewer 2: 1.4.0 should probably be `{{ full_version }}` (the syntax there is not correct, but it will expand to whatever is the current verrsion)

Reviewer: Yes, change it to this:
```suggestion
$ istioctl manifest apply --set installPackagePath=~/istio-releases/istio-{{< istio_full_version >}}/install/kubernetes/operator/charts
```

{{< /text >}}

> This will result in the same installation as `istioctl manifest apply` alone, because the command above points to the

Reviewer: ```suggestion
If using the `istioctl` {{< istio_full_version >}} binary, this command will result in the same installation as `istioctl manifest apply` alone, because it points to the
```

> same charts as the compiled in ones.

Reviewer: ```suggestion
same charts as the compiled-in ones.
```

> We recommend using the compiled-in charts rather than external ones because in this way the compatibility of the

Reviewer: ```suggestion
Other than for experimenting with or testing new features, we recommend using the compiled-in charts rather than external ones to ensure compatibility of the
```

> istioctl binary with the charts is guaranteed.

Reviewer: ```suggestion
`istioctl` binary with the charts.
```
